### PR TITLE
Updated link to widget options on Uploadcare

### PR DESCRIPTION
**Pull request by madebyreformat**
**Summary**

Theyve changed the URLs on the Uploadcare docs and i've updated the link to the widget options

**Test plan**

Not needed, just a change to a docs file

**A picture of a cute animal (not mandatory but encouraged)**

cute bat

### CODE REVIEW

File: website/content/docs/uploadcare.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ using the image or file widgets.
 ## Configuring the Uploadcare Widget
 
 The Uploadcare widget can be configured with settings that are outlined [in their
-docs](https://uploadcare.com/docs/uploads/widget/config/). The widget itself accepts configration
+docs](https://uploadcare.com/docs/file_uploads/widget/options/). The widget itself accepts configration
```
Comment (talves) [inline website/content/docs/uploadcare.md]: I think the equivalent would be:
https://uploadcare.com/docs/file_uploads/widget/config/

What do you think?

Comment (erquhart) [reply website/content/docs/uploadcare.md]: Uploadcare redirects to the url in this PR, and that URL does provide the list of configuration options, which is the intended target here.

**Comment by erquhart** [comment]
Thanks for the update!

**Comment by talves** [comment]
This link is incorrect.

**Comment by erquhart** [comment]
Why?

**Comment by talves** [comment]
See my preview. I supplied the link to the section it references.

**Comment by erquhart** [comment]
Your review just has the link you're proposing we use instead. Why do you prefer that link?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by madebyreformat** [comment]
It has the options for the widget rather than a general overview page

**Pull request merged**